final_task/rss_reader/rss_reader.py

```diff
@@ -0,0 +1,183 @@
+import argparse
+from typing import Union
+
+import feedparser
+import logging
+import json
+import html
+import sys
+import datetime
+import os
+
+__VERSION__ = '3.5'
+ARGS = None
+LIMIT = 3
+LINKS = []
+PRINT_SETTING = {
+    'feed': [
+        'title'
+    ],
+    'entries': [
+        'title',
+        'published',
+        'link',
+        'summary'
+    ]
+}
+
+
+def get_object_feed(url: str) -> Union[str, feedparser.FeedParserDict]:
+    try:
+        data = feedparser.parse(url)
+        if data.status == 200:
+            if data.bozo:
+                return f'ERROR: There is no rss feed at this url: {url}'
+            else:
+                return data
+        else:
+            return f'HTTP Status Code {data.status}'
+    except Exception as exc:
+        return f'ERROR: {exc}'
+
+
+def to_json(data) -> json:
+    """convert data to JSON format"""
+    result = {}
+    if isinstance(data, feedparser.FeedParserDict):
+        if 'feed' in PRINT_SETTING.keys():
```

Comment:
A question:

Why do you need this check if you yourself created this dict with this key at the top of the module?
Same question for check a bit lower.